bin/hdfcoinc/pycbc_multiifo_page_foreground

```diff
@@ -0,0 +1,161 @@
+#!/usr/bin/env python
+""" Make a table of found injection information
```

Comment:
Hi @GarethDaviesGW is this the code for making a table of found injections or is it a table of coincident multi-ifo foreground coincident events.
```suggestion
""" Make a table of coincident multi-ifo foreground coincident events.
"""
```

Reply:
Good point (you can see from this that I used the injection table as a basis for the new code!)